extensions/comment/models/Comment.php

```diff
@@ -50,7 +51,7 @@ public function rules()
             [
                 'verifyCode',
                 'captcha',
-                'captchaAction' => 'captcha/captcha',
+                'captchaAction' => Url::to(['captcha']),
```

Comment:
its ok but by this, each controller that has attached comment must have captcha action.
i think its better to have captcha action in this extension and attach it to CommentController and set its url in this validator